### ISSUE TITLE
Use standard keys for cut/copy/paste/new/close

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mintty/mintty#252]
```
I would like to be able to use standard Windows keys for cut (Ctrl-X), copy 
(Ctrl-C), paste (Ctrl-V) and creating new windows (Ctrl-N) and closing windows 
(Ctrl-W).

This would make it easier to copy text from other lines and paste it onto a new 
line.  It would also cut down on the number of new keystrokes that a new user 
needs to remember in order to use the terminal.


```

Original issue reported on code.google.com by `phidia...@gmail.com` on 10 Mar 2011 at 1:34

**Comment by mintty** [comment]
Released 2.9.1.

**Comment by mintty** [comment]
Released 3.6.0.
